### PR TITLE
Add continuous deployment to Dash server

### PR DESCRIPTION
**Pull request by tamidodo**
I've passed along the required secrets to the ALS team to add to the github repo, otherwise should be good to go. I've added a new user for LBL to access our services server and given that user access to the "prod" environment of our prototype app.

### CODE REVIEW

File: .github/workflows/staging-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        with:
+          python-version: '3.9.16'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Lint with flake8
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 pages app.py utils index.py predeploy_update_data.py tasks.py --count --select=E9,F63,F7,F82 --show-source --statistics
```
Comment (hannahker) [inline .github/workflows/staging-deploy.yml:23]: Can remove these specific files and just run `flake8` against all python files in the repo

**Review by hannahker — COMMENTED** [review]
Looks great! Just see `flake8` comment. Can you also add in the missing `Procfile` for a DE deploy?

File: .github/workflows/staging-deploy.yml (hunk 1 above)
Comment (tamidodo) [reply .github/workflows/staging-deploy.yml:23]: oops yup

**Review by hannahker — APPROVED** [review]
Looks great! 💃

**Pull request merged**